install/install.ps1

```diff
@@ -39,28 +38,18 @@ if (Test-Path -Path $extractedDirectory) {
     Remove-Item -Force -Recurse $extractedDirectory
 }
 
-if (Test-Path -Path $targetDirectory) {
-    Write-Verbose "Removing pre-existing target directory at ${targetDirectory}"
-    Remove-Item -Force -Recurse $targetDirectory
-}

```

Comment:
Why do we not still need to remove the `$extractDirectory` if it already exists?

Reply:
We still do remove the extracted directory. This was removing the `$targetDirectory`, which was `v0.1.0`/`v0.2.0`, etc.